devicetypes/smartthings/ezex-smart-occupancy-sensor.src/ezex-smart-occupancy-sensor.groovy

```diff
@@ -0,0 +1,106 @@
+/*
+ *  Copyright 2019 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy
+ *  of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ *  Author : Fen Mei / f.mei@samsung.com
+ *  Date : 2019-02-12
+ */
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+import physicalgraph.zigbee.zcl.DataType
+metadata {
+    definition(name: "eZEX Smart Occupancy Sensor(AC Type)", namespace: "smartthings", author: "SmartThings", runLocally: false, mnmn: "SmartThings", vid: "SmartThings-smartthings-ZigBee_Dimmer_with_Motion_Sensor") {
+        capability "Motion Sensor"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Health Check"
+        capability "Sensor"
+       
+        fingerprint profileId: "0104", deviceId: "0107", inClusters: "0000, 0003, 0004, 0406", outClusters:"0006, 0019", manufacturer:"", model:"E280-KR0A0Z0-HA", deviceJoinName: "eZEX Smart Occupancy Sensor(AC Type)"
+    }
+    tiles(scale: 2) {
+        multiAttributeTile(name: "motion", type: "generic", width: 6, height: 4) {
+            tileAttribute("device.motion", key: "PRIMARY_CONTROL") {
+                attributeState "active", label: 'motion', icon: "st.motion.motion.active", backgroundColor: "#00A0DC"
+                attributeState "inactive", label: 'no motion', icon: "st.motion.motion.inactive", backgroundColor: "#cccccc"
+            }
+        }
+
+        standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+            state "default", action: "refresh.refresh", icon: "st.secondary.refresh"
+        }
+        main(["motion"])
+        details(["motion", "refresh"])
+    }
+}
+
+def installed(){
+    log.debug "installed"
+    return zigbee.readAttribute(0x0406, 0x0000)
```

Comment:
`Installed()` will not actually send the command to read the attribute. This should be wrapped in a `response()` call.

Even better would be to refactor this method to simply be:

`response(refresh())`

and also change `ping()` to just call `refresh()` to avoid unnecessary code and magic number duplication.